### PR TITLE
Header

### PR DESCRIPTION
**Pull request by nadiacollado**
- [x] **User can see a website header**
- [x] **User can click on a menu to open and close it** 

- Adds GeneApp, Header components
- Tests functionality for GeneApp and Header
- Removes code from create-react-app

### CODE REVIEW

File: react-playground/src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import styled from "styled-components";
+import MainLogo from "../images/main-logo.svg";
+import MenuIcon from "../images/menu-icon.svg";
+
+const HeaderWrapper = styled.div`
```
Comment (nadiacollado) [inline react-playground/src/components/Header.tsx:6]: I imagine the styled components are meant to go in a separate file. Do you create files for each individual component or do they live together?

File: react-playground/src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  "Home",
+  "My Results",
+  "Health Profile",
+  "Community",
+  "Resources",
+  "Settings",
+  "Log out",
+];
+
+function Header() {
```
Comment (nadiacollado) [inline react-playground/src/components/Header.tsx:55]: Wondering how I'm meant to incorporate Typescript for this project. So far I haven't used it very much.

File: react-playground/src/components/Header.tsx (hunk 1 above)
Comment (proxycase) [reply react-playground/src/components/Header.tsx:6]: Yeah that makes sense. Usually we do a `src/components/Header/Header.tsx` type of ordering. We also have an `src/app/components` for more platform specific components as well, but the former option should work for your case.

Comment (proxycase) [reply react-playground/src/components/Header.tsx:6]: Inside of that Header/ folder, beside the component, we also include an index.ts with `export { default as Card } from './Card';` to enable simpler import statements when you need the component, like: `import { Header } from 'components/Header`

Comment (proxycase) [reply react-playground/src/components/Header.tsx:6]: You can reference the starter kit for some examples, though that project doesn't use the relative imports/exports idea there (the index files) https://github.com/8thlight/react-ui-starter-kit/tree/main/src/components

File: react-playground/src/components/Header.tsx (hunk 2 above)
Comment (proxycase) [reply react-playground/src/components/Header.tsx:55]: Go ahead and copy the Card component into your project from the react ui starter kit for a baseline typescript implementation. You may be able to also leverage that component for the background container for the main menu

**Review by proxycase — APPROVED** [review]
looks good, just a few ideas you can implement in this pr or later.

**Review by ranizilpelwar — APPROVED** [review]
Approved, loved Hugh's comments! +1 on them.

**Pull request merged**